src/domains/transaction/i18n.ts

```diff
@@ -384,6 +384,9 @@ export const translations: { [key: string]: any } = {
 	},
 
 	VALIDATION: {
-		LOW_BALANCE: "The balance it too low ({{balance}} {{ coinId }})",
+		EMPTY_BALANCE: "The balance is empty",
+		LOW_BALANCE: "The balance is too low ({{balance}} {{ coinId }})",
+		FEE_BELOW_MINIMUM: "The selected fee is below the minimum  ({{balance}} {{ min }})and may not be forged",
+		FEE_ABOVE_MAXIMUM: "The selected fee is above the maximum ({{balance}} {{ max }})",
```

Comment:
Why we need to display the balance over here?

Reply:
I remember changing that to be ({{min/max}} {{ coinId }}) but maybe broke after some rebasing. It' wasn't me. It was Patricia. Will add ({{min/max}} {{ coinId }}) for now until Sam's review of all these texts and re-adjust. :+1: 

Thanks for taking the time for this review. :myman